### PR TITLE
Refs #3809 - Enable cop StringConversionInInterpolation

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by lzap** [comment]
Rebase pls

**Comment by dLobatog** [comment]
@lzap Rebased!

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ Lint/ParenthesesAsGroupedExpression:
 # Offense count: 20
 # Cop supports --auto-correct.
 Lint/StringConversionInInterpolation:
```
Comment (domcleal) [inline .rubocop_todo.yml]: Doesn't this block get removed from the todo file now?

Comment (lzap) [inline .rubocop_todo.yml]: Yeah delete.

**Comment by dLobatog** [comment]
@lzap @domcleal Sorry for the leftover TODO, I've rebased and removed it. Hope it's all good now.

**Comment by dLobatog** [comment]
Rebased again after Rubocop was updated to 0.28.0

**Comment by lzap** [comment]
Tests failed, [test].

**Comment by domcleal** [comment]
It needs rebasing for tests to run.

**Comment by lzap** [comment]
Sorry @eLobato I was busy last couple of days. Rebase and I will merge.

**Comment by dLobatog** [comment]
@lzap no problem, I've rebased it but tests won't run until https://github.com/theforeman/foreman/pull/2115 fixes the current issue.

**Comment by lzap** [comment]
Let's just wait :-)

**Comment by lzap** [comment]
Okay [test] !

**Comment by lzap** [comment]
I see a regression with Ruby 2.0.0. http://ci.theforeman.org/job/test_develop_pr_core/3783/database=postgresql,ruby=2.0.0/

**Comment by domcleal** [comment]
@lzap that's a different, older job...

**Comment by lzap** [comment]
Hmmm [test]

**Comment by domcleal** [comment]
@lzap looks like Katello's tests are suffering from some failures, best to ignore it

**Comment by lzap** [comment]
Yeah NIC code I assume, merged as 5c02cfe. Thanks for patience @eLobato !

**Pull request closed without merging**